lib/data/DBusers.dart

```diff
@@ -1,16 +1,16 @@
-import 'package:sciencenotes/domain/people.dart';
+import 'package:sciencenotes/domain/users.dart';
 
-class BD {
-  static List<People> list = [
-    People(
-      image:
-      'https://static.thenounproject.com/png/448730-200.png',
-      name: 'Julia Ferreira',
-      email: 'jfo3@aluno.ifal.edu.br',
-      password: '12345',
-      user: 'fjuliaaf',
-      birthdate: '21/04/2005',
-    ),
+class DB {
+  static List<Users> list = [
```

Comment:
Você está usando este arquivo em algum lugar? Praticamente todo ele está comentado, o que me leva a acreditar que ele está sendo usado apenas para suprir alguma pendência de outra tela que não diz respeito a você, mas que se for removido vai dar erro no código. Correto? Do contrário, acho interessante remover para o professor não pensar que não estamos utilizando 100% o banco de dados construído.